Sloth.Core/Domain/WebHookRequest.cs

```diff
@@ -30,10 +30,6 @@ public class WebHookRequest
 
         public bool Persist { get; set; }
 
-        public WebHookRequestResendJobRecord WebHookRequestResendJob { get; set; }
```

Comment:
Does this need to point to the common one?

Reply:
Oops. That one needs to be deleted. We're not going to have any foreign keys to the JobRecords table.